src/Network/Wai/Middleware/OpenApi/PathMap.hs

```diff
@@ -1,8 +1,10 @@
-module Network.Wai.Middleware.OpenApi.PathMap
-  ( PathMap
-  , fromOpenApi
-  , lookup
-  ) where
+module Network.Wai.Middleware.OpenApi.PathMap (
+  PathMap,
+  fromOpenApi,
+  fromList,
+  lookup,
+  empty,
+) where
```

Comment:
Whoops, thanks for adding the `fourmolu.yaml`. Can you re-run it to squash out these differences?